firebase-firestore/src/main/java/com/google/firebase/firestore/DocumentId.kt

```diff
@@ -38,7 +38,7 @@ import kotlinx.serialization.SerialInfo
 @SerialInfo
 @Retention(RetentionPolicy.RUNTIME)
 @Target(
-// TODO: revert this file back to Java or find out a method to share this file between Java and Ktx.
+    AnnotationTarget.PROPERTY,
     AnnotationTarget.FUNCTION,
     AnnotationTarget.PROPERTY_GETTER,
     AnnotationTarget.PROPERTY_SETTER,
```

Comment:
Do we really need these? I see they were removed from `ServerTimestamp`

Reply:
Sorry, I missed these two lines, I just removed them. We do not need these as Kotlin serialization annotations on setter and getter will be transparent to the compiler plugin. Thank you.